src/zzz_od/api/routers/battle_assistant.py

```
@@ -0,0 +1,2509 @@
+        raise
+    except Exception as e:
+        raise HTTPException(
+            status_code=500,
+            detail={
+                "error": {
+                    "code": "CONFIG_UPDATE_FAILED",
+                    "message": f"更新指令调试配置失败: {str(e)}"
+                }
+            }
+        )
+        _assistant_config.operation_debug_repeat_mode = config.repeat_mode
```

Ruff (0.12.2)

Undefined name `_assistant_config`

(F821)